src/pages/RealtimeMapPage.tsx

```diff
@@ -47,7 +47,7 @@ export function numberToColorHsl(i: number, max: number) {
 }
 
 const fiveMinutesAgo = moment().subtract(5, 'minutes')
-const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')
+const fourMinutesAgo = moment().subtract(4, 'minutes')
```

Comment:
did you try revert this change? @amabelleS

Reply:
> did you try revert this change? @amabelleS

I'm not sure what you mean.. my code is, from the last PR: 
```const fiveMinutesAgo = moment().subtract(5, 'minutes')
const fourMinutesAgo = moment().subtract(4, 'minutes')```

this line: ```const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')``` is from the version before my last PR. 
There was a bug, in the initial render fourMinutesAgo & fiveMinutesAgo both got the value of 4 minutes ago. 
I did not revert it:( If that means that I chose to reintroduce this bug.

I'm a GitHub merge conflicts newbie:) relatively...

Reply:
I believe the `const fourMinutesAgo = fiveMinutesAgo.add(1, 'minutes')` is better, and asking you to revert the change you're making here.
When you construct a new `Moment`, it can make a little gap in time, so the difference won't be exactly one minute.
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/d26993ee-179e-4fcb-81c1-5b46e7e32b11)

Reply:
```
const fiveMinutesAgo = moment().subtract(5, 'minutes')
const fourMinutesAgo = moment(fiveMinutesAgo).add(1, 'minutes')
```
This fixes this bug.